src/components/Hub/HubFilters.tsx

```diff
@@ -21,49 +15,48 @@ type HubFilterProps = {
 export const getSelectedFilters = (filters: Filter[]): FilterParams => {
   return {
     kind: getCheckedFilterValues(filters[0]),
-    keywords: getCheckedFilterValues(filters[1]),
+    keywords: [...getCheckedFilterValues(filters[1]), ...getCheckedFilterValues(filters[2]), ...getCheckedFilterValues(filters[3]),],
```

Comment:
If the array has a fixed length, it should be a touple type, right? https://www.typescriptlang.org/docs/handbook/basic-types.html#tuple

Reply:
It could also be an empty array before it is set.